### PR TITLE
80104 workaround for if user is not directly invited to fusion meeting

### PR DESCRIPTION
**Pull request by bratli**
DevOps#80104

Workaround so that users getting IPO that are not invited to meeting can get all details, except for outlook response.  Add StartTime, EndTime, and Location to IPO db

### CODE REVIEW

File: src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/IPOContextModelSnapshot.cs [hunk 1]
```diff
@@ -15,16 +15,16 @@ protected override void BuildModel(ModelBuilder modelBuilder)
...
-                .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+                .HasAnnotation("ProductVersion", "5.0.0");
 
             modelBuilder.Entity("Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate.Attachment", b =>
                 {
                     b.Property<int>("Id")
                         .ValueGeneratedOnAdd()
                         .HasColumnType("int")
-                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+                        .UseIdentityColumn();
```
Comment (bratli) [inline src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/IPOContextModelSnapshot.cs:27]: @hbjorgo jeg har ikke gjort noe her selv, har det skjedd noe feil eller er denne filen grei?

File: src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs [hunk 2]
```diff
@@ -41,15 +41,25 @@ public async Task<Result<InvitationDto>> Handle(GetInvitationByIdQuery request,
...
+            GeneralMeeting meeting;
+            try
             {
-                throw new Exception($"Could not get meeting with id {invitation.MeetingId} from Fusion");
+                meeting = await _meetingClient.GetMeetingAsync(invitation.MeetingId,
+                    query => query.ExpandInviteBodyHtml().ExpandProperty("participants.outlookstatus"));
+
+                if (meeting == null)
+                {
+                    return new NotFoundResult<InvitationDto>($"Could not get meeting with id {invitation.MeetingId} from Fusion");
```
Comment (hbjorgo) [inline src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs:57]: If there's an exception, we continue, but with an empty meeting we return? Is that correct?

**Review by hbjorgo — APPROVED** [review]

File: src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs (hunk 2 above)
Comment (bratli) [reply src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs:57]: This is correct (from the information I have now). The reason:
If meeting == null, this means that the meeting does not exist in fusion at all, and something has gone wrong :(

If we get an exception, then it means that we got some other error, most likely the person is not directly invited to the meeting.

Do you think we should keep getting IPO even if meeting is null, or keep it like it is?

Comment (bratli) [reply src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs:57]: I will change it so that we are not dependant on Fusion!

**Pull request merged**